system/coldboxModifications/RequestContextDecorator.cfc

```diff
@@ -358,7 +358,11 @@ component accessors=true extends="preside.system.coldboxModifications.RequestCon
 
 	public boolean function showNonLiveContent() {
 		try {
-			return request._showNonLiveContent;
+			if ( this.isAdminUser() ) {
+				return true;
```

Comment:
This negates the purpose of the feature. As an admin, you can choose whether or not to show non-live content in the Preside admin bar (when viewing the frontend).

The issue here is that when the _default_ is set to _not_ show non-live content in `Config.cfc`, it starts also effecting the backend (i.e. data manager)